pages/partials/instructorInfoPanel.ejs

```diff
@@ -69,26 +69,38 @@
   <% } %>
   <% } %>
 
-  <% if (typeof assessment !== "undefined") { %>
+  <% let instructorUrlPrefix = urlPrefix;
+     let ip_assessment = null;
+     let ip_assessment_instance = null;
+     if (typeof assessment !== "undefined") {
+        ip_assessment = assessment;
+        ip_assessment_instance = assessment_instance;
+        instructorUrlPrefix = urlPrefix + '/instructor';
+     }
+     if (typeof variant !== "undefined" && variant.assessment) {
+        ip_assessment = variant.assessment;
+        ip_assessment_instance = variant.assessment_instance;
```

Comment:
Could we separately test for `variant.assessment_instance` to be robust in the future if we have variants associated with assessments but not assessment instances?

Reply:
> Could we separately test for `variant.assessment_instance` to be robust in the future if we have variants associated with assessments but not assessment instances?

That would probably rely on a different view anyway. I'll change the logic to use `assessment_instance` (the other conversion) instead of assessment, and leave the view of having variants linked to assessments when they come.